content/influxdb/v2.0/reference/cli/influxd/upgrade.md

```diff
@@ -36,23 +36,24 @@ influxd upgrade [command]
-|      | `--v1-dir`                      | Path to source 1.x `db` directory containing `meta`, `data`, and `wal` sub-folders (default `~/.influxdb`) | string     |
-|      | `--v2-config-path`              | Destination path for upgraded 2.x configuration file (default `~/.influxdbv2/config.toml`)                 | string     |
-| `-v` | `--verbose`                     | Verbose output                                                                                             |            |
+| Flag |                                  | Description                                                                                                | Input type |
+|:-----|:------------------------         |:-----------------------------------------------------------------------------------------------------------|:----------:|
+| `-m` | `--bolt-path`                    | Path for boltdb database (default `~/.influxdbv2/influxd.bolt`)                                            | string     |
+| `-b` | `--bucket`                       | Primary bucket name                                                                                        | string     |
+|      | `--config-file`                  | Custom InfluxDB 1.x config file path (default `~/.influxdb/influxdb.conf`)                                 | string     |
+|      | `--continuous-query-export-path` | Path for exported 1.x continuous queries (default `~/continuous_queries.txt`)                              | string     |
+| `-e` | `--engine-path`                  | Path for persistent engine files (default `~/.influxdbv2/engine`)                                          | string     |
+| `-f` | `--force`                        | Skip the confirmation prompt                                                                               |            |
+| `-h` | `--help`                         | Help for `influxd upgrade`                                                                                 |            |
+| `-c` | `--influx-configs-path`          | Path for 2.x `influx` CLI configurations file (default `~/.influxdbv2/configs`)                            |            |
+|      | `--log-level`                    | Supported log levels are `debug`, `info`, `warn`, and `error` (default `info`)                             | string     |
+|      | `--log-path`                     | Custom log file path (default `~/upgrade.log`)                                                             | string     |
+| `-o` | `--org`                          | Primary organization name                                                                                  | string     |
+|      | `--overwrite-existing-v2`        | Overwrite existing files at the output path instead of aborting the upgrade process                        |            |
```

Comment:
```suggestion
|      | `--overwrite-existing-v2`        | Overwrite existing files at the specified output path instead of aborting the upgrade process                        |            |
```

Reply:
Is output path `influx-configs-path`?

Reply:
No, the output path is where all the influxdb files live. There isn't an option to specify that specifically. I had included a specific option, but Dan pointed out it was incorrect.

Reply:
How does InfluxDB detect the default output path? Would it be helpful to clarify where default is?

Reply:
There isn't a singular output path (which is what makes making a one-liner description here difficult). The full list of options that set outputs is:

* `bolt-path`
* `continuous-query-export-path`
* `influx-configs-path`
* `engine-path`
* `v2-config-path`

I would change this to:
```suggestion
|      | `--overwrite-existing-v2`        | Overwrite existing files at the output paths instead of aborting the upgrade process                        |            |
```
For 2.0.5 I can look into making the help-text print options in labeled groups, so we can have an "outputs" group to make this more obvious.